eo-runtime/src/test/java/EOorg/EOeolang/EOtryTest.java

```diff
@@ -86,25 +86,6 @@ public void usesCatcherOutput() {
         );
     }
 
-    @Test
```

Comment:
@c71n93 why was this test removed?

Reply:
there was 2 same tests with different names